include/kframework/evm.md

```diff
@@ -284,6 +286,17 @@ OpCode Execution
  // -----------------------------------------------------------------------------------------------
     rule #lookupOpCode(BA, I, SCHED) => #dasmOpCode(BA[I], SCHED) requires 0 <=Int I andBool I <Int lengthBytes(BA)
     rule #lookupOpCode(_, _, _)  => .NoOpCode [owise]
+
+    syntax MaybeOpCode ::= "#unconsOpCode" "(" Bytes "," Schedule ")" [function, total]
+ // -------------------------------------------------------------------------------------------
+    rule #unconsOpCode(BA, SCHED) => #dasmOpCode(BA[0], SCHED) requires 0 <Int lengthBytes(BA)
+    rule #unconsOpCode(_, _)      => .NoOpCode                 [owise]
+
+    syntax Bytes ::= #dropOpCode(Bytes, MaybeOpCode)  [function, total]
+ // --------------------------------------------------------------
+    rule #dropOpCode(BA, _)         => .Bytes                                         requires lengthBytes(BA) ==Int 0
+    rule #dropOpCode(BA, .NoOpCode) => BA
+    rule #dropOpCode(BA, OP)        => substrBytes(BA, #widthOp(OP), lengthBytes(BA)) [owise]
```

Comment:
You'll need to make a special case for `PUSH(N)` opcodes. The arguments for `PUSH(...)` are directly inline in the bytecode, so for example `PUSH(1) 80` is encoded in the bytecode as `0x6080`, and `PUSH(2) 80` is encoded in the bytecode as `0x610080`.

So you need to special-case the `PUSH(N)` opcodes in these methods, to select more than just the first byte (or drop more than just the first byte).

You may also consider `head(...)` and `tail(...)` as the names of these?